Source/WebCore/platform/ReferrerPolicy.h

```diff
@@ -51,7 +51,7 @@ enum class ReferrerPolicy : uint8_t {
 };
 
 enum class ReferrerPolicySource : uint8_t { MetaTag, HTTPHeader, ReferrerPolicyAttribute };
-std::optional<ReferrerPolicy> parseReferrerPolicy(StringView, ReferrerPolicySource);
+std::optional<ReferrerPolicy> parseReferrerPolicy(const String&, ReferrerPolicySource);
```

Comment:
This is a string regression, why do we want to switch from StringView to `const String&` and force call sites to convert their StringView into a String, just to get a enum back?

Reply:
Yeah, I should have used toStringWithoutCopying, but even then the case sensitive JS comparison code is not really compatible, so I removed that part from my new patch.